### PR TITLE
Style

### PR DESCRIPTION
**Pull request by ch2ch3**
It's Hacky McHackerson 🎩 but good enough for now!

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
```
Comment (bkach) [inline frontend/public/index.html:7]: To take up the whole screen i suppose?

File: frontend/src/components/App/component.js [hunk 2]
```diff
@@ -3,7 +3,9 @@ import React from 'react';
 import GameList from 'components/GameList';
 import PlayerList from 'components/PlayerList';
 
-const App = ({ game, player }) => {
+import styles from './styles.module.css';
+
+const Router = ({ game, player }) => {
```
Comment (bkach) [inline frontend/src/components/App/component.js:8]: Router! Smart.

File: frontend/src/components/App/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.root {
+  max-width: 800px;
```
Comment (bkach) [inline frontend/src/components/App/styles.module.css:2]: Any reason for this number?

File: frontend/src/components/List/index.js [hunk 4]
```diff
@@ -1,22 +1,34 @@
...
+            className={cx(styles.button, styles.content)}
+            onClick={() => handleSelect(item)}
+          >
+            {item.name}
+          </button>
+          <button
+            className={cx(styles.button, styles.control, styles.delete)}
+            onClick={() => handleDelete(item.id)}
+          >
+            &#10799;
```
Comment (bkach) [inline frontend/src/components/List/index.js:23]: Unicode ✨

File: frontend/public/index.html (hunk 1 above)
Comment (ch2ch3) [reply frontend/public/index.html:7]: Yep on smaller screens

File: frontend/src/components/App/component.js (hunk 2 above)
Comment (ch2ch3) [reply frontend/src/components/App/component.js:8]: "Router" 😅

File: frontend/src/components/App/styles.module.css (hunk 3 above)
Comment (ch2ch3) [reply frontend/src/components/App/styles.module.css:2]: Nah, it's pretty arbitrary for now and will likely change when we start displaying the board!

File: frontend/src/components/List/index.js (hunk 4 above)
Comment (ch2ch3) [reply frontend/src/components/List/index.js:23]: 🦄

**Pull request merged**